build.rs

```diff
@@ -46,6 +56,9 @@ pub fn copy<U: AsRef<Path>, V: AsRef<Path>>(from: U, to: V) -> Result<(), std::i
                 match path.file_name() {
                     Some(filename) => {
                         let dest_path = dest.join(filename);
+                        if overwrite_behaviour == OverwriteBehaviour::ClobberReadOnlyFiles {
+                            fs::remove_file(&dest_path).ok();
+                        }
```

Comment:
```suggestion
                        // Some `git` files are created with write protection, so replacing them
                        // directly can fail with a permissions error. Remove the destination file
                        // first. Ignore any errors, only the fs::copy() call is critical.
                        fs::remove_file(&dest_path).ok();
```

I've realized we only call `copy()` from one place, always with `OverwriteBehaviour::ClobberReadOnlyFiles`, so I don't think that pulls the weight of introducing an enum and a paremeter -> I propose to simply hardcode the behavior. This is a single-purpose build script, so this can be arbitrarily specific.

If we fear this behavior is surprising, we can document that in `copy()`'s docstring.

Reply:
Agreed, when I saw the bool parameter I didn't realize it was only called once. The `pub` marking also threw me off.

Reply:
Yeah, I was thrown off by the pub too 😅 I agree about making the behaviour inconditional.

Reply:
I went ahead and pushed this change.